.github/workflows/test-e2e-semgrep-ci.jsonnet

```diff
@@ -162,7 +161,7 @@ local semgrep_ci_fail_open_blocking_findings_job = {
         if semgrep ci --suppress-errors; then
            exit 2
         else
-           exit 0
+           exit 3
```

Comment:
FYI this is just to test this PR; when you active the workflow, it should now fail (because of the exit 3)
and trigger the if failure and slack notification.